app/models/GQLSchema.scala

```diff
@@ -763,9 +791,44 @@ object GQLSchema extends GQLMeta with GQLEntities {
           val entities = ctx.arg(entityNames).getOrElse(Seq.empty)
           ctx.ctx.search(ctx.arg(queryString), ctx.arg(pageArg), entities)
         }),
-      Field("associationsOnTheFly", associationsObTheFlyGQLImp,
-        Some("associations on the fly"),
-        resolve = ctx => ctx.value)
+
+//      Field("aotfByDisease", associatedOTFTargetsImp,
```

Comment:
This commented out code can be removed.

Reply:
Yes, I missed it as I left in there while I was refactoring